packages/core/upload/admin/src/utils/normalizeAPIError.ts

```diff
@@ -0,0 +1,103 @@
+import type { errors } from '@strapi/utils';
+import type { FetchError } from '@strapi/admin/strapi-admin';
+
+type ApiError =
+  | errors.ApplicationError
+  | errors.ForbiddenError
+  | errors.NotFoundError
+  | errors.NotImplementedError
+  | errors.PaginationError
+  | errors.PayloadTooLargeError
+  | errors.PolicyError
+  | errors.RateLimitError
+  | errors.UnauthorizedError
+  | errors.ValidationError
+  | errors.YupValidationError;
```

Comment:
I'm a bit concerned this list could get outdated. maybe something more programmatic could work? I haven't tried it but something along the lines of `type ApiError = errors[keyof typeof errors]`

Reply:
done